### DISCUSSION TITLE
Block Applications with Signing Date Less Than X

### DISCUSSION THREAD

**Discussion by dsbaha** [northpolesec/santa#172] [Ideas]
Basically; if the software signing date is less than X, block the application.
Background; If I have Google Chrome blanket allowed by Team ID, and I don't want Google Chrome < v130 running, the thought is to block on an old signing date.  I didn't see in the 'codesign' utility where applications have a version number.

**Comment by russellhancox** [comment]
https://northpole.dev/cookbook/cel/#apps-signed-since-x
